stellar/deploy-contract.js

```diff
@@ -53,27 +53,25 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
     }
 }
 
-async function processDeploy(options, config, chain) {
+async function deploy(options, config, chain) {
     const { contractName, privateKey, wasmPath } = options;
     const { rpc, networkType } = chain;
     const networkPassphrase = getNetworkPassphrase(networkType);
     const wallet = await getWallet(chain, options);
 
     if (!chain.contracts) {
         chain.contracts = {};
+        return;
```

Comment:
why return? this condition is initializing the contracts key if it didn't exist

Reply:
It was added for testing; should be removed.